### ISSUE TITLE
Add already ported Python 3 modules to porting howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5956]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#5956|
|Type|task|
|Created|2012-09-07 13:13:30Z|
|Branch|https://github.com/twisted/twisted/tree/more-python3-porting-guide-5956|

We should document API differences/pecularities in already ported modules in the Python 3 porting howto.

<details><summary>Searchable metadata</summary>

```
trac-id__5956 5956
type__task task
reporter__itamar itamar
priority__normal normal
milestone__Python_3_3_Minimal Python 3.3 Minimal
branch__branches_more_python3_porting_guide_5956 branches/more-python3-porting-guide-5956
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1347023610000000 1347023610000000
changetime__1349376843000000 1349376843000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @itamarst|
|-|-|

I sort of punted, maybe it's good enough.  Build results: http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/coding-standard-py3-5939

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

Sorry.  Wrong ticket.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

Hmm.  I wish this ticket had a list of what modules were in that "already ported" list. :/

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @itamarst|
|-|-|

I just looked at what we've ported already and tried to catch changes.  Found a few.  We should try to be careful going forward to keep this porting guide up to date, rather than forgetting and then have to go back and figure out what changed.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

(In [35937]) Branching to 'more-python3-porting-guide-5956'

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_6"></a>|@itamarst set owner to @exarkun|
|-|-|

Arguably the fact that `FilePath` stayed as bytes is worth nothing, even if in theory nothing changed (consider all the unit tests we needed to update), so might be worth mentioning. And yeah, we should make sure changes end up here. Add mention of `FilePath`, then merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

(In [35953]) Note FilePath

refs [#5956](https://github.com/twisted/twisted/issues/5956)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [35954]) Merge more-python3-porting-guide-5956

Author: exarkun
Reviewer: itamar
Fixes: [#5956](https://github.com/twisted/twisted/issues/5956)

Add some notes about potential traps to application developers who are porting
a Twisted-based application to Python 3.
